### DISCUSSION TITLE
66

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#792] [Comments]
http://jojoldu.tistory.com/66

**Comment by jojoldu** [comment]
작성자: Kingbbode

크게 배우고 갑니다. 감사합니다.

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

앗 ㅋㅋ kingbbode님! 봐주셔서 감사합니다 
오히려 제가 더 배우는게 많습니다 
항상 문제해결하는 방법을 보고 감탄 많이 하고 있습니다^^

**Comment by jojoldu** [comment]
작성자: 지나가는개발자

좋은글 써주셔서 감사합니다 ^*^ 좋은하루되세요~

**Comment by eggrok** [comment]
점수를 계산할때, A카드의 경우는 1 또는 11로 계산이 되는데. 그런부분은 빠진건가요?
